src/plot/series/mark-series-gl.js

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import {PropTypes} from 'react';
+import {ScatterplotLayer, COORDINATE_SYSTEM} from 'deck.gl';
+import {rgb} from 'd3-color';
+
+import {DEFAULT_SIZE, DEFAULT_OPACITY} from 'theme';
+import {getAttributeFunctor} from 'utils/scales-utils';
+
+import AbstractSeries from './abstract-series';
+
+class MarkSeriesGL extends AbstractSeries {
+  static get requiresSVG() {
+    return false;
+  }
+
+  static get isDeckGL() {
+    return true;
+  }
+
+  static renderLayer(props) {
+    const {
+      data,
+      _renderKey,
+      seriesId,
+      colorDomain,
+      colorRange,
+      sizeDomain,
+      sizeRange,
+      fp64
+    } = props;
+
+    const xFunctor = getAttributeFunctor(props, 'x');
```

Comment:
This is also a bit out of scope but for posterity...

Not sure if I'd call this a `functor` unless I wanted to evoke its meaning from category theory (and the corresponding fantasy-land spec for JS). I know this is what devs in C++ call function objects, but that terminology isn't really useful in JS since we already have first-class functions.